src/redux/ui-notifications/types.ts

```diff
@@ -30,7 +29,7 @@ export interface DispatchOptions {
 export interface UiNotification extends DispatchOptions {
   titleI18nKey: string
   contentI18nKey: string
-  date: DateTime
+  date: number
```

Comment:
Maybe rename this to timestamp to clarify the kind of value.
A date given as number could be interpreted in multiple ways.

(This and all other places with date as variable name)